ui/app/src/views/SinglePool.vue

```diff
@@ -4,28 +4,36 @@ import { computed, toRefs } from "@vue/reactivity";
 import Layout from "@/components/layout/Layout.vue";
 import SifButton from "@/components/shared/SifButton.vue";
 import { getAssetLabel, useAssetItem } from "@/components/shared/utils";
-import { Fraction, LiquidityProvider, Pool, usePoolCalculator, AssetAmount, Asset } from "ui-core";
+import { Fraction } from "ui-core";
 import { useCore } from "@/hooks/useCore";
-import { useRoute } from 'vue-router';
-import { getBlockExplorerUrl } from "../components/shared/utils"
+import { useRoute } from "vue-router";
+import { getBlockExplorerUrl } from "../components/shared/utils";
 
-const DECIMALS = 5
+const DECIMALS = 5;
```

Comment:
@59023g @ryardley I believe we should do 6 as a minimum. We show 6 in many places, and only in pools/liquidity part we show 5 for some reason

Reply:
Good idea but not sure that is part of this mandate though?